//test error

void foo() {}

int main() {
  int x = 7;
  void t;
  return x;
}
